domains/certificates/Query.API/API/Program.cs

```diff
@@ -34,18 +46,29 @@
 builder.Logging.AddSerilog(console.CreateLogger());
 
 builder.Services.AddOpenTelemetry()
-    .WithMetrics(provider =>
-        provider
+    .ConfigureResource(resource => resource
+        .AddService(serviceName: "Query.API"))
```

Comment:
I tried before, with no luck. I will try again

Reply:
I remember now, why it wasn't possible.

If i were to fetch the name dynamically from the Assembly, it would be named "API". This isn't very informative, since we also have services named "API", in Auth, Transfer, "Measurements", subsystems, so that is why i am doing it statically.